pkg/localConfigProvider/localConfigProvider.go

```diff
@@ -0,0 +1,55 @@
+package localConfigProvider
+
+// URLKind is an enum to indicate the type of the URL i.e ingress/route
+type URLKind string
+
+const (
+	DOCKER  URLKind = "docker"
+	INGRESS URLKind = "ingress"
+	ROUTE   URLKind = "route"
+)
+
+// LocalURL holds URL related information
+type LocalURL struct {
```

Comment:
I'm confused as to why we're defining this struct here. `localConfigProvider` package is meant for providing a component to deal with s2i components or something else? It seems dumb on my part to not know what `localConfigProvider` is about. :disappointed:

Reply:
Moved it to a separate package for future purpose in case we define a new local config type

Reply:
> Moved it to a separate package

But I'm still seeing it at the same place in the `localConfigProvider.go` file. What am I missing here? :thinking:

Reply:
I meant I moved the interface and the `LocalURL` struct into a separate package in case a new type of local config provider comes up in the future.